aries-site/src/components/cards/CardGrid.js

```diff
@@ -21,7 +21,11 @@ export const CardGrid = ({ cards, ...rest }) => {
         cards.map(topic => (
           <Link key={topic.name} href={nameToPath(topic.name)} passHref>
             <ContentCard
-              as="a"
+              /* forwardedAs maintains styling provided by Higher Order 
+              Components. Styled-Components 4.3.0 docs: https://styled-components.com/docs/api#forwardedas-prop
```

Comment:
Is it still relevant on styled-component 5.0.0?

Reply:
It is currently listed in the Styled Component docs.